examples/artemis-multicast-producer/artemis_multicast_producer.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/artemis;
+import ballerina/log;
+
+// Creates a producer to the address "my_address"
+artemis:Producer prod = new({host:"localhost", port:61616}, "my_address",
+  addressConfig = {routingType:artemis:MULTICAST});
+public function main() {
+    // Sends the string message to the artemis server
```

Comment:
```suggestion
    // Sends the message as a String to the Artemis server.
```

Reply:
It is really a string type message so saying it as string message makes more sense.